packages/design-system/src/components/Accordion/Accordion.tsx

```diff
@@ -11,6 +11,10 @@ export interface AccordionProps {
    * Class to be applied to the outer `<div>` that contains all accordion items.
    */
   className?: string;
+  /**
+   * Use alternate Accordion icon and styles in place of the standard styles
+   */
+  isAlternateStyles?: boolean;
```

Comment:
Because we don't necessarily want this style in the core design system yet, I think we could make the customization here a bit more generic by replacing this `isAlternateStyles` prop with props for controlling the icon that is being used, like `openIcon` and `closeIcon`. The default value for core would be
```
openIcon: AddIcon,
closeIcon: RemoveIcon,
```
for healthcare, instead of [exporting a new `Accordion` component](https://github.com/CMSgov/design-system/pull/1913/files#diff-c2f3e80e3ecfe87e59fb8e4a2457574a1abdd75211a51305cad6f4395f3add80R6), you could just import the core accordion and override its defaults like
```tsx
Accordion.defaultProps = {
  openIcon: PlusCircleIcon,
  closeIcon: MinusCircleIcon,
};
```
and then you don't have to re-export anything.

Reply:
You would then want to move the alt styles that are in `packages/design-system/src/styles/components/_Accordion.scss` to `packages/ds-healthcare-gov/src/styles/components/_Accordion.scss` and override the core `.ds-c-accordion` rules.